robocorp-code/src/robocorp_code/robocorp_language_server.py

```diff
@@ -825,7 +822,7 @@ def _compute_new_output_workitem_path(
 
         next_run = max_run + 1
         self._prefix_to_last_run_number_and_time[output_prefix] = (
-            next_run,
```

Comment:
This isn't a real fix as 2 runs may end up running with the same id if a run is done before the next run creates the folder related to the output.

Rather, the proper fix is changing this function to accept a parameter for a case where the output shouldn't be incremented (the issue is that there are calls done to this function which shouldn't increment this number).